src/common/factory.ts

```diff
@@ -39,6 +64,8 @@ export abstract class TurboBaseFactory {
   }: TurboUnauthenticatedConfiguration = {}) {
     token = token === 'pol' ? 'matic' : token;
 
+    token ??= 'arweave'; // default to arweave if token is not provided
```

Comment:
it works with parenthesis around (token ??= 'arweave'), but doesn't appear to provide type safety on token being defined

Reply:
the 'pol' convenience part I think could be expanded to include an "acceptableTokenStringMap" that converts the tickers as well like

```
sol => solana
eth => ethereum
ar => arweave
```

it would be a littel easier to use the token input parameter on the sdk, but no need for yet